roles/mirror_ocp_release/tasks/main.yml

```diff
@@ -78,4 +84,9 @@
   ansible.builtin.include_tasks: registry.yml
   when:
     - mor_mirror_container_images | bool
+
+- name: Copy artifacts to release directory
+  ansible.builtin.copy:
+    src: "{{ _mor_tmp.path }}/"
+    dest: "{{ mor_cache_dir }}/{{ mor_version }}"
```

Comment:
At the end of the job execution the artifacts are copied to the job's target directory. For this, we use a regular task at the end of the role instead of handlers, so we don't wait for the end of the play to copy the artifacts.